src/lib/common/globals.h

```diff
@@ -32,7 +32,11 @@
 #include "common/Timer.h"
 #include "common/sem.h"
 
-
+/* ****************************************************************************
+*
+* FIXME P10: temporal definition, it should be removed soon
+*/
+#define PARANOID_JSON_INDENT
```

Comment:
Ah, it will be removed?
So, perhaps this PR should include the removal of the define ... ?

Reply:
No, this PR cannot include that, but a soon one will do. See the plan in https://github.com/telefonicaid/fiware-orion/issues/2760#issuecomment-337334053.

Reply:
Ok

NTC